server.js

```diff
@@ -1,7 +1,25 @@
+require('dotenv').config();
 var express = require('express')
 var app = express()
+const port = 5000;
+const mongoose = require('mongoose');
+
+const uri = 'mongodb+srv://ishita_naraniya:ishhMongodb@cluster1.ponxmip.mongodb.net/?retryWrites=true&w=majority';
+app.get('/', (req, res) => {
+    mongoose.connect(uri)
```

Comment:
Consider handling the database connection in a separate module or function. This would make the code more modular and easier to maintain. You could then import and use this function in your server.js file. [medium]